### ISSUE TITLE
Epic account not authenticated

### ISSUE THREAD

**Issue by Kinc4id** [Lacro59/playnite-successstory-plugin#500]
Since the latest update I always get the error epic account not authenticated. When I go to the settings it says it is authenticated. I authenticated anyways and still get the error after reboot.

**Comment by Kinc4id** [comment]
I don't know if it's related but also since yesterday success story takes a very long time upon start up and then shows an error 503 service unavailable.

**Comment by Lacro59** [comment]
Can you post your extension.log?

**Comment by Kinc4id** [comment]
https://paste.ee/p/nhEgC

**Comment by pem884** [comment]
Perhaps relevant? I was experiencing this, too. It seemed to be resolved for a while, then I received an email yesterday:

> You’re receiving this email because we noticed suspicious activity on your Epic Games account. Most often, the cause of this type of activity is the result of using the same password on multiple websites, use of a weak password, or phishing.

I should add here that I don't suggest there is anything nefarious going on with the extension; just that perhaps they have some measures in place which - maybe - could complicate things for some users.

After receiving the email I saw the extension had been logged out / deauthorized / etc. from Epic, which I expected as the email stated it would happen. I authorized, which went pretty smoothly after one or two tries, and I don't believe I am having any problems.

TL;DR: I am not having any issues, though I originally did a few weeks ago. Epic thinks something strange is happening with my account. I signed back in and it's still working OK as far as I can tell. Others may want to check their email for notifications to see whether this applies to you.

**Comment by Bovive** [comment]
> After receiving the email I saw the extension had been logged out / deauthorized / etc. from Epic, which I expected as the email stated it would happen. I authorized, which went pretty smoothly after one or two tries, and I don't believe I am having any problems.


This is happening to me too. I have reset my password twice now because of it. I just disabled Epic in SuccessStory for now.

**Comment by CJohnnyB** [comment]
I had the same problem.

At the same time, not sure if related Some games are importing achievements and others are not.
for example, I have Death Stranding and is not importing achievements at all. I know that the achievements are tracked, as I saw them in Epic and also I see Exophase importing them in my profile. So something is not right on how SuccessStory is collecting them.

**Comment by Lacro59** [comment]
https://github.com/Lacro59/playnite-successstory-plugin/commit/9ec0dc984a8852d871127f54f4117a945e4e8c39
https://github.com/Lacro59/playnite-plugincommon/commit/d7e39e075afa06e7b0c46497fcafce2e8532346d
https://github.com/Lacro59/playnite-plugincommon/commit/ac332e128a041a7e9cc9bdf92ab6c6cd199ab5e7

**Comment by Kinc4id** [comment]
If this means it should be fixed it isn't for me. I'm still not authenticated after retart and updating achievements times out.

**Comment by Lacro59** [comment]
@Kinc4id, can you post your extension.log?

**Comment by Lacro59** [comment]
Is the issue still present?

**Comment by CJohnnyB** [comment]
In my case I don't see login problems anymore, but achievements progress is not being imported for Epic games at all.

I mean I see the game achievements, but not the ones I have unlocked.

**Comment by veksha** [comment]
@CJohnnyB 
> I mean I see the game achievements, but not the ones I have unlocked.

try the following:

**Change privacy level to public then back to private through the web site.**

1. go to https://store.epicgames.com/en-US/ (important. epic games launcher can't change privacy. bug?)
2. press on your nickname -> My Achivements -> Privacy level (click on dots "...") -> Manage privacy.
3. change to Public. Try to fetch achivements in Playnite (also turn "Is private" option off  in extension settings)
4. change back to "Friends of Friends" Try to fetch achivements in Playnite (also turn "Is private" option on  in extension settings)

for some reason now it works for me.

**Comment by CJohnnyB** [comment]
Amazing! that did it...

Thanks a lot @veksha

**Comment by Lacro59** [comment]
Can you try this version?
[playnite-successstory-plugin_3_4_1.zip](https://github.com/user-attachments/files/19760576/playnite-successstory-plugin_3_4_1.zip)
